tests/labhub_test.py

```diff
@@ -180,6 +180,25 @@ def test_assign_cmd(self):
 
         testbot.assertCommand(cmd.format('coala', 'a', '23'),
                               'You\'ve been assigned to the issue')
+        
+        # no assignee, newcomer, initiatives/gci
+        mock_maint_team.is_member.return_value = False
+        mock_dev_team.is_member.return_value = False
+        mock_issue.labels = 'initiatives/gci',
+        mock_issue.assignees = tuple()
+        testbot.assertCommand(cmd.format('coala', 'a', '23'),
+                              'not eligible to be assigned to this issue')
+        testbot.pop_message()
+
+        # no assignee, developer, initiatives/gci
+        mock_maint_team.is_member.return_value = False
+        mock_dev_team.is_member.return_value = True
+        mock_issue.labels = 'initiatives/gci',
+        mock_issue.assignees = tuple()
+        testbot.assertCommand(cmd.format('coala', 'a', '23'),
+                              'not eligible to be assigned to this issue')
```

Comment:
Where is the `You are ...` in the message? :) I would compare against the full message, so nobody asks you why you are asserting for just a part of the message (like me) ;)

Reply:
@Makman2 Added the 'You are ...' :smiley: